plugins/aver-hid/fu-aver-hid-device.c

```diff
@@ -271,13 +309,31 @@ fu_aver_hid_device_write_firmware(FuDevice *device,
 	fu_progress_add_step(progress, FWUPD_STATUS_DEVICE_BUSY, 1, NULL);
 	fu_progress_add_step(progress, FWUPD_STATUS_DEVICE_WRITE, 44, NULL);
 	fu_progress_add_step(progress, FWUPD_STATUS_DEVICE_VERIFY, 15, NULL);
-	fu_progress_add_step(progress, FWUPD_STATUS_DEVICE_WRITE, 30, NULL);
+	fu_progress_add_step(progress, FWUPD_STATUS_DEVICE_WRITE, 15, NULL);
+	fu_progress_add_step(progress, FWUPD_STATUS_DEVICE_WRITE, 15, NULL);
 
 	/* get default image */
 	fw = fu_firmware_get_bytes(firmware, error);
 	if (fw == NULL)
 		return FALSE;
 
+	/* decompress twice */
```

Comment:
a zip in a zip?

Reply:
Yes.
Each SoC has a compressed file, and then all the compressed files will be combined into a single compressed fw file.
So there has one compressed file in a zip file for the devices with one SoC. And there have two compressed files in a zip file for the devices with two SoCs.